src/utils/rest-api.ts

```diff
@@ -220,12 +220,19 @@ export function deleteElement(elementUuid: string) {
     });
 }
 
-export function deleteElements(elementUuids: string[], activeDirectory: string) {
+export async function deleteElements(elementUuids: string[], activeDirectory: string) {
     console.info('Deleting elements : %s', elementUuids);
-    const idsParams = getRequestParamFromList('ids', elementUuids).toString();
-    return backendFetch(PREFIX_EXPLORE_SERVER_QUERIES + `/v1/explore/elements/` + activeDirectory + '?' + idsParams, {
-        method: 'delete',
-    });
+    const chunkSize = 50;
```

Comment:
I think this should be extracted as a constant, especially if this behaviour is supposed to be applied elsewhere (cf https://github.com/gridsuite/commons-ui/pull/598)